src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I like how variables and methods are named well which improves the readability of the code and reduces bugs